src/js/components/Grid/doc.js

```diff
@@ -54,13 +54,22 @@ space in the column axis.`,
       'around',
       'stretch',
     ]).description('How to align the contents along the column axis.'),
-    areas: PropTypes.arrayOf(
-      PropTypes.shape({
-        name: PropTypes.string,
-        start: PropTypes.arrayOf(PropTypes.number),
-        end: PropTypes.arrayOf(PropTypes.number),
-      }),
-    ).description('Area names and column,row coordinates.'),
+    areas: PropTypes.oneOfType([
+      PropTypes.arrayOf(
+        PropTypes.shape({
+          name: PropTypes.string,
+          start: PropTypes.arrayOf(PropTypes.number),
+          end: PropTypes.arrayOf(PropTypes.number),
+        }),
+      ),
+      PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.string)),
+    ]).description(
+      `Grid areas.
```

Comment:
Another suggestion for the description:

'Either area names and column,row coordinates. Or, an array of string arrays that specify named grid areas.'

Feel free to refine or make any other suggestions.

Reply:
Done